server/app/lib/analysis_library/schema/osa.json

```diff
@@ -0,0 +1,1106 @@
+					"type": "number",
+                    "default": 66,
+                    "minimum": 1
+				},
+                "nboot": {
+                    "description": "Sobol: The number of bootstrap replicates",
+					"type": "number",
+                    "default": 0,
+				},
+                "norm_type": {
+                    "description": "fast99, Ga, GaIsl, Morris, optim, PSO, RGenoud, Sobol, SPEA2",
+					"type": "string",
+                    "default": "minkowski",
+					"enum": [
+						"minkowski", "maximum", "euclidean", "binary", "manhattan"
+					]
+				},
+                "normtype": {
+					"type": "string",
+					"enum": [
+						"minkowski", "maximum", "euclidean", "binary", "manhattan"
+					]
+				},
+                "npart": {
+					"description": "PSO: Number of particles in the swarm",
+					"type": "number",
+                    "default": 4,
+                    "minimum": 1
+				},
+                "number_of_runs": {
+					"description": "Number of runs.",
+					"type": "number",
+                    "minimum": 1
+				},
+                "number_of_samples": {
+					"description": "Number of samples.",
+					"type": "number",
+                    "minimum": 1
+				},
+                "numIslands": {
+					"description": "GaIsl: Integer value specifying the number of islands to be used in a ring topology in which each island is connected unidirectionally with another island",
+					"type": "number",
+                    "default": 4,
+                    "minimum": 0
+				},
+                "migrationRate": {
+					"description": "GaIsl: Value in range 0-1 providing the proportion of individuals that should migrate between the islands",
+					"type": "number",
+                    "default": 0.1,
+                    "minimum": 0,
+                    "maximum": 1
+				},
+                "migrationInterval": {
+					"description": "GaIsl: Integer value specifying the number of iterations at which exchange of individuals takes place",
+					"type": "number",
+                    "default": 10,
+                    "minimum": 0
+				},
+                "objective_functions": {
+					"description": "List of the objective functions",
+					"type": "array",
+					"items": {
+						"type": "string"
+					}
+				},
+                "order": {
+                    "description": "Sobol: the maximum order in the ANOVA decomposition (all indices up to this order will be computed)",
+					"type": "number",
+                    "default": 1,
+				},
+                "pgtol": {
+                    "description": "Optim, RGenoud: Tolerance on the projected gradient",
+					"type": "number",
+                    "default": 0.1,
+                    "minimum": 0
+				},
+                "popsize": {
+                    "description": "RGenoud: Size of initial population",
+					"type": "number",
+                    "default": 2,
+                    "minimum": 1
+				},
+                "popSize": {

```

Comment:
is popsize and popSize different?

Reply:
they world be since json is case sensitive.